test/units/module_utils/aws/test_rds.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+# (c) 2017, Michael De La Rue
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+
+from ansible.module_utils.aws.rds import get_rds_db_instance, rds_instance_to_facts, get_rds_snapshot, rds_snap_to_facts, rds_facts_diff
+import unittest
+import datetime
+import copy
+from dateutil.tz import tzutc
+
+
+class FakeResource():
```

Comment:
maybe;  I'm hand checking to ensure no private data leaks out.  If someone could lend me access to an official Ansible account that is set up so that data can clearly be publicised that would make using placebo etc. easier.